File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewModelMyStandardControlsRow.cs
/// <filename>
///     RowBasedCommand_ViewModelMyStandardControlsRow.cs
/// </filename>


namespace commands
{
    public class RowBasedCommand_ViewModelMyStandardControlsRow
  {
        private string TextBoxesTextBoxText;
     
        public     virtual string getTextBoxesTextBoxText()
    {
      return this.TextBoxesTextBoxText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
     
        private int RowDepth;
     
        public     virtual int getRowDepth()
    {
      return this.RowDepth;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewModelMyStandardControlsRow.cpp

#include "RowBasedCommand_ViewModelMyStandardControlsRow.h"
#include <string>

namespace commands
{
  std::string RowBasedCommand_ViewModelMyStandardControlsRow::getTextBoxesTextBoxText( ) 
  {
    return this->TextBoxesTextBoxText;
  }
  int RowBasedCommand_ViewModelMyStandardControlsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
  int RowBasedCommand_ViewModelMyStandardControlsRow::getRowDepth( ) 
  {
    return this->RowDepth;
  }
}

